sdk/storage/storage-file-datalake/test/node/pathclient.spec.ts

```diff
@@ -311,6 +311,16 @@ describe("DataLakePathClient Node.js only", () => {
     await destFileClient.getProperties();
   });
 
+  it("move should encode source", async () => {
+    const destFileName = recorder.getUniqueName(" a+'%20%2F%2B%27%%25%2520.txt");
```

Comment:
Should we add another case for special chars file name without url encoded?

Reply:
This string contains both url encoded and not-encoded char like space. I think it's enough.